src/invidious/helpers/helpers.cr

```diff
@@ -64,10 +64,12 @@ end
 class Config
   include YAML::Serializable
 
-  property channel_threads : Int32                 # Number of threads to use for crawling videos from channels (for updating subscriptions)
-  property feed_threads : Int32                    # Number of threads to use for updating feeds
+  property channel_threads : Int32 = 1             # Number of threads to use for crawling videos from channels (for updating subscriptions)
+  property feed_threads : Int32 = 1                # Number of threads to use for updating feeds
+  property output : String = "STDOUT"              # Log file path or STDOUT
+  property log_level : LogLevel = LogLevel::Info   # Default log level, see src/invidious/helpers/logger.cr
```

Comment:
An option like `log_level: info` would be better than `log_level: LogLevel::Info`